src/netplan.c

```diff
@@ -500,6 +500,24 @@ write_dhcp_overrides(yaml_event_t* event, yaml_emitter_t* emitter, const char* k
 err_path: return FALSE; // LCOV_EXCL_LINE
 }
 
+STATIC gboolean
+write_ipv6_ra_overrides(yaml_event_t* event, yaml_emitter_t* emitter, const char* key, const NetplanNetDefinition* def, const NetplanIPv6RAOverrides* data)
+{
+    if (DIRTY_COMPLEX(def, *data)
+        || !data->use_dns
+        || data->use_domains
+        || data->route_table != NETPLAN_ROUTE_TABLE_UNSPEC) {
+        YAML_SCALAR_PLAIN(event, emitter, key);
+        YAML_MAPPING_OPEN(event, emitter);
+        YAML_BOOL_FALSE(def, event, emitter, "use-dns", data->use_dns);
+        YAML_STRING_PLAIN(def, event, emitter, "use-domains", data->use_domains);
+        YAML_UINT_DEFAULT(def, event, emitter, "route-table", data->route_table, NETPLAN_ROUTE_TABLE_UNSPEC);
+        YAML_MAPPING_CLOSE(event, emitter);
+    }
+    return TRUE;
+err_path: return FALSE; // LCOV_EXCL_LINE
```

Comment:
This could be removed as it's not used.

Reply:
Regarding this, the `YAML_MAPPING_CLOSE` macro uses `err_path`, soo I wasn't able to remove this line without breaking it